include/mbedtls/ssl.h

```diff
@@ -1427,6 +1427,7 @@ struct mbedtls_ssl_config
                               *   If either no PSK or a raw PSK have been
```

Comment:
Note: this comment needs updating now that `mbedtls_ssl_conf_psk()` is storing things here too.

Reply:
Also adding a comment on `psk_opaque_is_internal` in the same time